includes/services/OpenLayers/jquery.openlayers.js

```diff
@@ -100,6 +100,7 @@
 				markerData.lonlat.transform(new OpenLayers.Projection("EPSG:4326"), new OpenLayers.Projection("EPSG:900913"));
 			}
 
+			if (this.bounds != null) this.bounds.extend(markerData.lonlat); // Extend the bounds when no center is set.
```

Comment:
I guess. I tried to provoke something like this with the SemanticMaps ajax feature (which only loads and adds points that are within current bounds), but it worked fine.